scripts/install.js

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env node
+
+"use strict";
+const _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) { return typeof obj; } : function (obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj; };
```

Comment:
I would suggest moving `const _typeof` and `validateConfiguration` function to a separate file too, since it makes visual noice (the typeof) and have same implementation across the files. It can be imported with `require` in this scripts. This sepearate file must also be published.